sdk/servicebus/azure-messaging-servicebus/docs/SyncReceiveAndPrefetch.md

```diff
@@ -10,7 +10,7 @@ Even after the application disables prefetch in the builder, the `receiveMessage
 IterableStream<ServiceBusReceivedMessage> receiveMessages(int maxMessages, Duration maxWaitTime)
 ```
 
-where `maxMessages` is the maximum number of messages to receive and `maxWaitTime` is the overall duration the SDK should wait for the entire `maxMessages` to arrive.
+where `maxMessages` is the maximum number of messages to receive and `maxWaitTime` is the maximum duration client should wait for entire batch of `maxMessages` messages to arrive.
```

Comment:
> should wait for entire batch of `maxMessages` messages to arrive.
Missing the word 'the':  Should be "should wait for **the** entire batch of `maxMessages` messages to arrive."